sumdig_i.rb

```diff
@@ -0,0 +1,15 @@
+def sumdig_i(n)
+	remaining_portion = n
+	result = 0
+	while remaining_portion > 9
```

Comment:
If you change the `while` condition to `while remaining_portion > 0`, you can skip this line: `result += remaining_portion` and return `result` immediately when the loop exits.